### PR TITLE
Suggested Edit

### PR DESCRIPTION
**Pull request by jimmychu0807**
@sacha-l  Further links fixing

### CODE REVIEW

File: v3/docs/02-concepts/b-extrinsics/index.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ keywords:
 An extrinsic is a piece of information that comes from outside the chain and is included in a block.
 Extrinsics fall into three categories: inherents, signed transactions, and unsigned transactions.
 
-Note that [events](/v3/runtime/events) are not extrinsics. The chain emits events for pieces of
+Note that [events](../../runtime/events-and-errors) are not extrinsics. The chain emits events for pieces of
```
Comment (sacha-l) [inline v3/docs/02-concepts/b-extrinsics/index.mdx:13]:
```suggestion
Note that [events](/v3/runtime/events-and-errors) are not extrinsics. The chain emits events for pieces of
```

File: v3/docs/03-runtime/c-metadata/index.mdx [hunk 2]
```diff
@@ -453,8 +453,8 @@ the [`InvalidSpecName` error](/rustdocs/latest/frame_system/pallet/enum.Error.ht
 - [Storage](../storage)
 - [SCALE](/v3/advanced/scale-codec)
 - [Macros](../macros)
-- [Events](../events)
-- [Extrinsics](/v3/concepts/extrinsics)
+- [Events](../events-and-errors)
+- [Extrinsics](../../concepts/extrinsics)
```
Comment (sacha-l) [inline v3/docs/03-runtime/c-metadata/index.mdx:457]: @jimmychu0807 and here too?

**Review by sacha-l — APPROVED** [review]

File: v3/docs/02-concepts/b-extrinsics/index.mdx (hunk 1 above)
Comment (jimmychu0807) [reply v3/docs/02-concepts/b-extrinsics/index.mdx:13]: Yea, either way will works. Same as below.

File: v3/docs/03-runtime/c-metadata/index.mdx (hunk 2 above)
Comment (jimmychu0807) [reply v3/docs/03-runtime/c-metadata/index.mdx:457]: either one will work.

**Pull request merged**